src/lib/_common.scss

```diff
@@ -160,6 +160,7 @@ $header-height: 48px;
         display: flex;
         flex: 1 1 auto;
         align-items: center;
+        justify-content: center;
```

Comment:
What is the reason for this change? It seems pretty risky.

Reply:
> What is the reason for this change? It seems pretty risky.

so we can avoid using `left: 50%; transform: translateX(-50%);`